.vitepress/config.ts

```diff
@@ -686,34 +610,124 @@ function sidebarHome() {
                         },
                       ],
                     },
+                    
                   ],
                 },
+                {
+                  text: "Ethereum fallback mechanism",
+                  link: "/developers/ethereum-fallback",
+                },
+              ],
+            },
+            {
+              text: "Sovereign",
+              collapsed: true,
+                items: [
+                { text: "Rollkit", link: "https://rollkit.dev" },
+                {
+                  text: "Sovereign SDK",
+                  link: "https://github.com/Sovereign-Labs/sovereign-sdk/tree/stable/examples/demo-rollup#demo-rollup",
+                },
               ],
             },
-            { text: "Rollkit", link: "/developers/rollkit" },
             {
-              text: "Astria",
+              text: "Other",
+              collapsed: true,
+              items: [
+                {
+                  text: "Dymension",
+                  link: "https://dymension.xyz/",
```

Comment:
```suggestion
                  link: "https://docs.dymension.xyz/build/overview/",
```

Changing the link here @alex-beckett